factory/processComponents.go

```diff
@@ -424,12 +427,16 @@ func (pcf *processComponentsFactory) Create() (*processComponents, error) {
 		return nil, err
 	}
 
+	if err != nil {
+		return nil, err
```

Comment:
unreachable code due to L426-L428 (duplicated code here)

Reply:
:man_facepalming: